packages/suite-native-atoms/src/Select/SelectItem.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactNode } from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from '../Box';
+import { Radio } from '../Radio';
+import { Text } from '../Text';
+
+export type SelectItemProps = {
+    label: string;
+    value: string | number;
```

Comment:
create some type alias like `SelectValue` and use it everywhere

Reply:
[c4bbb5d](https://github.com/trezor/trezor-suite/pull/5506/commits/c4bbb5d664ce069fc6198695d54e89ad7df975ce)